### PR TITLE
[RTM] Auto-clear the session form data

### PR DESCRIPTION
**Pull request by leofeyer**
See #1389.

### CODE REVIEW

File: src/Resources/config/listener.yml [hunk 1]
```diff
@@ -41,6 +41,12 @@ services:
             # and lower than the header replay header (defaults to 7)
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 6 }
 
+    contao.listener.clear_form_data:
+        class: Contao\CoreBundle\EventListener\ClearFormDataListener
+        tags:
+            # The priority must be lower than the one of the Symfony save session listener (defaults to -1000)
```
Comment (aschempp) [inline src/Resources/config/listener.yml]: the priority must be **higher** than...

Comment (leofeyer) [reply src/Resources/config/listener.yml]: Fixed in 6847201f50a61d2fc69a8907d67074a7377d4396.

**Comment by aschempp** [comment]
To make it all work, do we need to start the session when form data is submitted and actually store the data?

**Comment by leofeyer** [comment]
Right now the session is started automatically thanks to our `LazySessionAccess` class. And since using `$_SESSION` is deprecated anyway, I would leave it this way.

**Pull request merged**